src/main/kotlin/badasintended/slotlink/init/BlockEntityTypes.kt

```diff
@@ -22,11 +26,14 @@ object BlockEntityTypes : Initializer {
     override fun main() {
         MASTER = r(B.MASTER, ::MasterBlockEntity)
         REQUEST = r(B.REQUEST, ::RequestBlockEntity)
+        INTERFACE = r(B.INTERFACE, ::InterfaceBlockEntity)
 
         CABLE = r(B.CABLE, ::CableBlockEntity)
         LINK_CABLE = r(B.LINK_CABLE, ::LinkCableBlockEntity)
         IMPORT_CABLE = r(B.IMPORT_CABLE, ::ImportCableBlockEntity)
         EXPORT_CABLE = r(B.EXPORT_CABLE, ::ExportCableBlockEntity)
+
+        ItemStorage.SIDED.registerForBlockEntity(InterfaceBlockEntity::getStorage, INTERFACE)
```

Comment:
You can use [`registerSelf`](https://github.com/FabricMC/fabric/blob/1f66e6ce26f5508775eaac8592d7074c1cd43c09/fabric-api-lookup-api-v1/src/main/java/net/fabricmc/fabric/api/lookup/v1/block/BlockApiLookup.java#L194) instead and remove the `getStorage` method.